apps/server/src/modules/class-booking/service.ts

```
@@ -0,0 +1,118 @@
+				.from("enrollments")
+				.select("*", { count: "exact", head: true })
+				.eq("cohort_id", cohort.id)
+				.in("status", ["paid", "welcome_package_sent"]);
+
+			if (countError) {
+				console.error("Error counting enrollments:", countError);
+				continue;
+			}
+
+			const activeEnrollments = count || 0;
+			const maxStudents = cohort.max_students || Infinity;
```

Biome (2.1.2)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)